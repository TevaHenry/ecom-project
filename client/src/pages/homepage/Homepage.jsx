import React from 'react';
import Directory from '../../components/Directory/Directory';
import { HomepageContainer } from './HompageStyles';

const Homepage = () => (
  <HomepageContainer>
    <Directory />
  </HomepageContainer>
)

export default Homepage;