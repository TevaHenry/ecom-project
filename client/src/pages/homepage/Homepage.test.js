import { shallow } from 'enzyme';
import React from 'react';
import Homepage from './Homepage';

it('expect to render Homepage component', () => {
  expect(shallow(<Homepage />)).toMatchSnapshot()
})