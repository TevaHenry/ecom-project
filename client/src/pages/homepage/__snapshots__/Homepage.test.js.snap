// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Homepage component 1`] = `ShallowWrapper {}`;
